tradeit-android-sdk/src/main/java/it/trade/android/sdk/manager/TradeItLinkedBrokerManager.java

```diff
@@ -75,20 +75,48 @@ public void onError(TradeItErrorResult error) {}
         });
     }
 
+    public void syncLinkedBrokers(List<TradeItLinkedLoginParcelable> linkedLoginParcelables) throws TradeItSaveLinkedLoginException, TradeItDeleteLinkedLoginException {
```

Comment:
This should be:
```java
public synchronized void syncLinkedBrokers(...) {
```

And eventually we should add an access lock to all operations on `this.linkedBrokers`:
```java
private final Object linkedBrokersLock = new Object();
...
public void someFunction() {
    synchronized (linkedBrokersLock) {
        // read or write to this.linkedBrokers
    }
}
```